src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -189,18 +190,17 @@ private String getAuthorizationHeader() {
 
     private Request getService(final String url, boolean isHttpGet) {
         String contentType = "application/" + ((isEncrypted) ? "jose+json" : "json");
-        if (isHttpGet) {
-            return new Request(url)
-                    .addHeader("Authorization", getAuthorizationHeader())
-                    .addHeader("Accept", contentType)
-                    .addHeader("User-Agent", "Hyperwallet Java SDK v" + version);
-        } else {
-            return new Request(url)
-                    .addHeader("Authorization", getAuthorizationHeader())
-                    .addHeader("Accept", contentType)
-                    .addHeader("Content-Type", contentType)
-                    .addHeader("User-Agent", "Hyperwallet Java SDK v" + version);
+        Request request = new Request(url)
+                .addHeader("Authorization", getAuthorizationHeader())
+                .addHeader("Accept", contentType)
+                .addHeader("User-Agent", "Hyperwallet Java SDK v" + version)
+                .addHeader("x-sdk-version", version)
+                .addHeader("x-sdk-type", "java")
+                .addHeader("x-sdk-contextId", String.valueOf(UUID.randomUUID()));
```

Comment:
The goal of the `x-sdk-contextId` is to track all requests that were made using the same Hyperwallet SDK instance, so I think will be better if we generate the randomUUID inside the constructor as a final variable.

Reply:
Done.